varia/rebuild-child-themes.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+declare -a ChildThemes=("alves" "balasana" "barnsbury" "brompton" "coutoire" "dalston" "exford" "hever" "leven" "mayland" "maywood" "morden" "redhill" "rivington" "rockfield" "shawburn" "stow" "stratford" )
```

Comment:
This is the file that does the magic. Did I miss any?

Reply:
Is it possible to explicitly select which themes to be recompile? 

Like if I wanted to just recompile **hever**, **alves**, **coutoire** and **dalston**, could I do that too?

Reply:
Sure, we could pass a param. Of course, if nothing changed in a child theme then the diff would be blank anyway...